packages/pymedphys_deliverydata/src/pymedphys_deliverydata/object/exportable.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (C) 2019 Cancer Care Associates
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version (the "AGPL-3.0+").
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License and the additional terms for more
+# details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# ADDITIONAL TERMS are also included as allowed by Section 7 of the GNU
+# Affero General Public License. These additional terms are Sections 1, 5,
+# 6, 7, 8, and 9 from the Apache License, Version 2.0 (the "Apache-2.0")
+# where all references to the definition "License" are instead defined to
+# mean the AGPL-3.0+.
+
+# You should have received a copy of the Apache-2.0 along with this
+# program. If not, see <http://www.apache.org/licenses/LICENSE-2.0>.
+
+
+import functools
+from typing import Union, Tuple
+
+
+from pymedphys_utilities.types import to_tuple
+from pymedphys_mudensity.mudensity import calc_mu_density
+
+from ..base import DeliveryDataBase
+from ..dicom import (
+    delivery_data_to_dicom,
+    dicom_to_delivery_data,
+    gantry_tol_from_gantry_angles)
+from ..utilities import (
+    filter_out_irrelevant_control_points,
+    get_all_masked_delivery_data,
+    get_metersets_from_delivery_data)
```

Comment:
Notice here how I am only ever importing from the module directories. I am trying to keep a separation of concerns here. Each directory module decides what API it will expose to the big bad world, that way, other modules in the same package are restricted to only using what that module is exposing.

Keeps a nice well defined API boundary.

Reply:
I see. I was wondering why these files were separated into subfolders. Now I know :)